beacon_node/beacon_processor/src/lib.rs

```diff
@@ -833,6 +843,8 @@ impl<E: EthSpec> BeaconProcessor<E> {
         // Using a FIFO queue since blocks need to be imported sequentially.
         let mut rpc_block_queue = FifoQueue::new(MAX_RPC_BLOCK_QUEUE_LEN);
         let mut rpc_blob_queue = FifoQueue::new(MAX_RPC_BLOB_QUEUE_LEN);
+        let mut rpc_verify_data_column_queue = FifoQueue::new(MAX_RPC_VERIFY_DATA_COLUMN_QUEUE_LEN);
+        let mut sampling_result_queue = FifoQueue::new(MAX_SAMPLING_RESULT_QUEUE_LEN);
```

Comment:
Looks like we never read / `pop` from these queues. We'll need to add them under this section:
https://github.com/sigp/lighthouse/blob/9de3178c7ffe6939210c643e28ec35d154d0c679/beacon_node/beacon_processor/src/lib.rs#L992

Reply:
Also worth adding the queue metrics so we don't forget:
https://github.com/sigp/lighthouse/blob/9de3178c7ffe6939210c643e28ec35d154d0c679/beacon_node/beacon_processor/src/lib.rs#L1420-L1423